test/parser-specs.js

```diff
@@ -83,6 +83,27 @@ describe('Server Parser', function () {
     parsed.allowInsecure.should.eql(['feature1', 'feature2', 'feature3']);
     parsed.denyInsecure.should.eql(['nofeature1', 'nofeature2', 'nofeature3']);
   });
+  it('should parse default driver args correctly from a string', function () {
+    let fakeArgs = `{"sillyWebServerPort":1234,"host":"hey"}`;
+    let fakeDriverName = `fake`;
+    let fakeDriverArgs = `{"${fakeDriverName}": ${fakeArgs}}`;
```

Comment:
i would probably compose all these as objects and then stringify in the parse invocation